test/functional/feature_llmq_is_retroactive.py

```
@@ -24,6 +24,16 @@ def set_test_params(self):
         # -whitelist is needed to avoid the trickling logic on node0
         self.set_dash_test_params(5, 4, [["-whitelist=127.0.0.1"], [], [], [], ["-minrelaytxfee=0.001"]])
 
+    # random delay before tx is actually send by network could take up to 30 seconds
+    def wait_for_tx(self, txid, node, expected=True, timeout=60):
```

Ruff (0.12.2)

Missing return type annotation for public function `wait_for_tx`

Add return type annotation: `None`

(ANN201)